content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,81 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl
+{{< tabs name="other_kubectl_uninstall" >}}
+
+{{% tab name="curl" %}}
+If you have used curl to install kubectl, then the uinstallation process can be done using :-
+
+```shell
+sudo rm -rf /usr/local/bin/kubectl
+sudo rm -rf /usr/local/lib/kubectl
+```
+
+An alternative command to carry out the uninstall:-
+
+```shell
+sudo rm $(which kubectl)
+```
+
+For getting binary files run the below command:
+
+```shell
+which kubectl
+```
+or
+```shell
+whereis kubectl
+```
+It is recommended to uninstall the executable binaries for completely removing the kubectl from the root.
+
+{{% /tab %}}
+
+{{% tab name="apt-get" %}}
+If you are on Ubuntu or another Debian based Linux distribution
+
+```shell
+sudo apt-get purge kubectl
+sudo apt-get autoremove
+```
+
+An alternative command to carry out the uninstall:-
+
+```shell
+sudo rm $(which kubectl)
+```
+
+The executable binaries are stored in the ```/usr/bin or usr/local/bin```, library files in ```usr/lib``` and configuration files in ```/etc```.
+It is recommended to manually remove the library files and static data for completely uninstalling the ```kubectl``` from the system.
+
+{{% /tab %}}
+
+{{% tab name="Homebrew" %}}
+If you are using homebrew as the package management tool on your machine, then the uninstallation process can be done using the below command:
+
+```shell
+sudo brew uninstall kubectl
+```
+
+The executable binaries are stored in the ```/usr/bin or usr/local/bin```, library files in ```usr/lib``` and configuration files in ```/etc```.
+It is recommended to manually remove the library files and static data before uninstalling the ```kubectl``` from the system.
```

Comment:
These steps may be unnecessary as the package manager will automatically find the path during the uninstallation process.